### PR TITLE
Have an opinion about using the broker.  Default to true.

### PR DESCRIPTION
**Pull request by AdamBJohnsonx**

### CODE REVIEW

File: msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java [hunk 1]
```diff
@@ -286,7 +286,7 @@ public Boolean getMultipleCloudsSupported() {
      * @return The boolean indicator of whether or not to use the broker.
      */
     public Boolean getUseBroker() {
-        return mUseBroker;
+        return mUseBroker == null ? true : false;
```
Comment (shahzaibj) [inline msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java:289]:
```suggestion
        return (mUseBroker == null) || mUseBroker;
```

Comment (shahzaibj) [reply msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java:289]: Suggested a change above, looks like we are ignoring the supplied value if it is not null

Comment (AdamBJohnsonx) [reply msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java]: You're right.  I wrote this while not paying attention.

**Review by shahzaibj — APPROVED** [review]

**Comment by AdamBJohnsonx** [comment]
This is likely a breaking change.  We should have taken this at 2.0.

**Pull request closed without merging**